File: LLM-Translations/GPT-4/codenet/Python/C/s906994739.c
#include <stdio.h>
#include <limits.h>

int main() {
    int n;
    scanf("%d", &n);

    int F[n][10];
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < 10; j++) {
            scanf("%d", &F[i][j]);
        }
    }

    int P[n][11];
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < 11; j++) {
            scanf("%d", &P[i][j]);
        }
    }

    int ans = INT_MIN;

    for (int i = 1; i < (1 << 10); i++) {
        int tmp = 0;
        int lst[n];
        for (int k = 0; k < n; k++) {
            lst[k] = 0;
        }

        for (int j = 0; j < 10; j++) { // time zone
            if ((i >> j) & 1) { // if open
                for (int k = 0; k < n; k++) {
                    lst[k] += F[k][j];
                }
            }
        }

        for (int k = 0; k < n; k++) {
            tmp += P[k][lst[k]];
        }

        ans = (ans > tmp) ? ans : tmp;
    }

    printf("%d\n", ans);

    return 0;
}
// 

File: LLM-Translations/GPT-4/codenet/Python/Go/s906994739.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)
	F := make([][10]int, n)
	P := make([][11]int, n)
	ans := -1e18

	for i := 0; i < n; i++ {
		for j := 0; j < 10; j++ {
			fmt.Scan(&F[i][j])
		}
	}
	for i := 0; i < n; i++ {
		for j := 0; j <= 10; j++ {
			fmt.Scan(&P[i][j])
		}
	}

	for i := 1; i < 1<<10; i++ {
		tmp := 0
		lst := make([]int, n)
		for j := 0; j < 10; j++ {
			if (i >> j) & 1 == 1 {
				for k := 0; k < n; k++ {
					lst[k] += F[k][j]
				}
			}
		}
		for k := 0; k < n; k++ {
			tmp += P[k][lst[k]]
		}
		if tmp > ans {
			ans = tmp
		}
	}

	fmt.Println(ans)
}
// 
